keyboards/blackplum/keymaps/default/keymap.c

```diff
@@ -0,0 +1,45 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
```

Comment:
This line is not actually needed, and should be removed. 

```suggestion
```

Reply:
Done